speech/google/cloud/speech/sample.py

```diff
@@ -95,10 +99,15 @@ def source_uri(self):
     def content(self):
         """Bytes of audio content.
 
-        :rtype: bytes
-        :returns: Byte stream of audio content.
+        :rtype: bytes or file
+        :returns: Byte stream of audio content or file like object.
         """
-        return self._content
+        if self._content is None:
+            return self._content
+        content = self._content
+        if getattr(self._content, 'read', None) is not None:
+            content = self._content.read()
+        return content
```

Comment:
You could shorten this up a lot. The entire body of this method / property can be

```python
try:
    return self._content.read()
except AttributeError:
    return self._content
```